src/main/java/io/vertx/core/http/HttpClientOptions.java

```diff
@@ -267,6 +272,7 @@
 
   private boolean shared;
   private String name;
+  private String pcapCaptureFile;
```

Comment:
this should be removed in favor of `NetworkOptions` settings

Reply:
Good point, fixed